content/docs/getting-started.md

```diff
@@ -9,17 +9,21 @@ weight: 10
 
 ## Building the dex binary
 
-Dex requires a Go installation and a GOPATH configured. For setting up a Go workspace, refer to the [official documentation][go-setup]. Clone it down the correct place, and simply type `make` to compile the dex binary.
+To build dex from source code, install a working Go environment with version 1.15 or greater according to the [official documentation][go-setup].
+Then clone the repository and use `make` to compile the dex binary.
 
 ```bash
-$ go get github.com/dexidp/dex
-$ cd $GOPATH/src/github.com/dexidp/dex
+$ mkdir -p $GOPATH/src/github.com/dexidp/
+$ cd $GOPATH/src/github.com/dexidp/
+$ git clone https://github.com/dexidp/dex.git
+$ cd dex
 $ make
 ```
+> **Note:** It is possible to clone the repository outside $GOPATH
```

Comment:
Noob question, and the answer may or may not need to be brought in if it's obvious to more seasoned dex devs, but what's the advantage to cloning the repo in the `GOPATH`?

Reply:
I see some benefits, but they are minor.
1. $GOPATH is probably exists
2. If you want to build an old version of a dex, you need to move the repository to $GOPATH

Reply:
I'd like to suggest then that we do remove the $GOPATH references, but potentially that we move them to an 'advanced' setup page or something like it? @sagikazarmark thoughts?